saleor/graphql/attribute/utils.py

```diff
@@ -534,6 +524,45 @@ def get_variant_selection_attributes(qs: "QuerySet") -> "QuerySet":
     )
 
 
+def prepare_attribute_values(
+    attribute: attribute_models.Attribute, attr_values: AttrValuesInput
+):
+    slug_to_value_map = {}
+    name_to_value_map = {}
+    for val in attribute.values.all():
```

Comment:
What if there are 1M attribute values? Think "artist" in a music store or "author" in a book store.